templates/openhub.html

```diff
@@ -38,6 +38,5 @@ <h1>All of our Portfolio Projects</h1>
       </div>
     </div>
     <hr>
-    {% endfor %}{# for project in projects #}
-  </body>
+    {% endfor %}{# for project in projects #} {# for project in projects #}  </body>
```

Comment:
`{# for project in projects #} ` have been repeated twice. This does not seem right. Please look into it.

Reply:
actually earlier i wrote it once but then gitmate was showing this error 
```
 gitmate-bot 15 hours ago Member

Unlabeled control end tag

Origin: Jinja2Bear, Section: all.jinja.

The issue can be fixed by applying the following patch:

--- a/tmp/tmptzzzojzz/templates/openhub.html
+++ b/tmp/tmptzzzojzz/templates/openhub.html
@@ -38,6 +38,6 @@
       </div>
     </div>
     <hr>
-    {% endfor %} {# for project in projects #}
+    {% endfor %}{# for project in projects #} {# for project in projects #}
   </body>
 </html>

```